test/test_suite.py

```diff
@@ -137,24 +137,24 @@ def test_smoke_test_suite_exists(self):
         assert isinstance(SmokeTestSuite, TestSuite)
         assert SmokeTestSuite.name == "smoke"
 
-    def test_smoke_test_suite_contains_relu(self):
+    def test_smoke_test_suite_contains_bmm(self):
         optests = list(SmokeTestSuite)
 
         assert len(optests) >= 1
-        assert optests[0].op == get_operator(torch.ops.aten.relu.default)
+        assert optests[0].op == get_operator(torch.ops.aten.bmm.default)
```

Comment:
fwiw i still buy that a simple pointwise op is probably a better test just cause you kinda just need to write some trivial triton code that most humans would know how to read.